raiden/ui/app.py

```diff
@@ -444,11 +445,17 @@ def run_app(
         except AddressWrongContract:
             handle_contract_wrong_address('secret registry', service_registry_contract_address)
 
-        config['services']['pathfinding_service_address'] = configure_pfs(
+        pfs_url, pfs_eth_address = configure_pfs(
             pfs_address=pathfinding_service_address,
+            pfs_eth_address=pathfinding_eth_address,
             routing_mode=routing_mode,
             service_registry=service_registry,
         )
+
+        assert pfs_eth_address is not None, 'Eth address of selected pathfinding service unknown.'
```

Comment:
nitpick: `is unknown`

Reminder to self: This will conflict with my startup changes: https://github.com/raiden-network/raiden/pull/3732